cigaradvisor/blocks/article-list/article-list.css

```diff
@@ -1 +1,54 @@
-/** Dummy File for now **/
+@import url('../article-teaser/article-teaser.css');
+
+.article-list.block .pagination-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-top: 30px;
+
+}
+
+.article-list.block .pagination {
+    display: flex;
+    padding-left: 0;
+    margin: 20px 0;
+    border-radius: 4px;
+}
+
+.article-list.block .pagination li {
+    display: inline;
+}
+
+.article-list.block .pagination a {
+    border: .5px solid var(--tan);
+    margin: auto 2px;
+    font-size: 15px;
```

Comment:
Variables here - if we don't have one, use the next size down.

We have variables for font size, weight, & line height

Reply:
fixed